spark/dl/src/test/scala/com/intel/analytics/bigdl/optim/DistriOptimizerSpec.scala

```diff
@@ -328,6 +326,7 @@ class DistriOptimizerSpec extends FlatSpec with Matchers with BeforeAndAfter {
   }
 
   "TrainSummary with MSE and LBFGS" should "work correctly" in {
+    TestUtils.cancelOnWindows()
```

Comment:
hadoop will try to call chmod underhood and it will cause a failure. So cancel it on windows